plugin/sycl/README.md

```diff
@@ -0,0 +1,37 @@
+<!--
+******************************************************************************
+* Copyright by Contributors 2017-2023
+*******************************************************************************/-->
+
+# SYCL-based Algorithm for Tree Construction
+This plugin adds support of SYCL programming model for prediction algorithms to XGBoost.
+
+## Usage
+Specify the 'device' parameter as described in the table below to offload model training and inference on SYCL device.
+
+### Algorithms
+| device | Description |
+| --- | --- |
+sycl | use default sycl device  |
+sycl:gpu | use default sycl gpu  |
+sycl:cpu | use default sycl cpu  |
```

Comment:
Should we add more explanations here on expectations? Like cpu at this point would launch sycl kernels on CPU device and this at this point functionally supported but no performance is expected in this case.

Reply:
added